assets/scss/_variables_project.scss

```diff
@@ -37,8 +37,17 @@ $table-row-alternate: $secondary-lightest-background;
 $table-row-default: $secondary-lighter-background;
 
 /*
- Opt to serve fonts locally by overriding web-font-path to be a non-google fonts URL.
- This is only possible with our modified docsy theme: https://github.com/matrix-org/docsy
+ Disable loading any fonts from Google Fonts.
```

Comment:
```suggestion
 * Configure docsy to use the default bootstrap fonts instead of Google Fonts.
 * See https://www.docsy.dev/docs/adding-content/lookandfeel/#fonts
```

Reply:
Note that no fonts are imported by bootstrap - bootstrap tries to load fonts from the system: https://getbootstrap.com/docs/5.2/content/reboot/#native-font-stack

`$td-enable-google-fonts` will disable fetching fonts from Google Fonts, and use system fonts instead. The list of system fonts is configured by `$font-family-sans-serif` below. The comment about "The list of fonts ..." tried to explain this, but I'll clarify it to mention `$font-family-sans-serif` explicitly. While the contents of `$font-family-sans-serif` looks like what bootstrap defines, it's originally from docsy's source code.

We have to override it (with Inter at the front) in order to load Inter.

Reply:
Done in a101159d96f684ae9b6563c4fb479ce81797b60e.

Reply:
Ok, the thing I really wanted to see here is a link to the place where `td-enable-google-fonts` is defined, which we still don't have. The interesting point here is that it's exposed by *docsy* rather than bootstrap.

I think my original suggestion is ok, with s/bootstrap/system/ ?

Reply:
Ah, yes we definitely should add a link to that piece of documentation.

> I think my original suggestion is ok, with s/bootstrap/system/ ?

Yes, this looks good on a re-read.

Reply:
Added the modified suggestion in 4473805c2743544912076162098a5e31666c8da0.